build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -64,3 +64,97 @@
 
 - name: Load Service Telemetry Operator CSV
   shell: oc apply -f working/service-telemetry-operator-bundle/manifests/service-telemetry-operator.clusterserviceversion.yaml -n "{{ namespace }}"
+
+# --- Loki Operator ---
+- name: Prevent Loki Operator from building operator-sdk
+  replace:
+    path: "{{ playbook_dir }}/working/loki-operator/.bingo/Variables.mk"
+    regexp: '^.*modfile=operator-sdk.mod.*$'
+    replace: ''
+
+- name: Prevent Loki Operator from putting authentication on /metrics
+  replace:
+    path: "{{ playbook_dir }}/working/loki-operator/config/overlays/openshift/kustomization.yaml"
+    regexp: '{{ item }}'
+    replace: ''
+  loop:
+    - "patchesStrategicMerge:"
+    - "- manager_auth_proxy_patch.yaml"
+    - "- manager_related_image_patch.yaml"
+    - "- manager_run_flags_patch.yaml"
+    - "- prometheus_service_monitor_patch.yaml"
+
+- name: Generate Loki Operator CSV
+  make:
+    chdir: "{{ playbook_dir }}/working/loki-operator"
+    target: bundle
+    params:
+        REGISTRY_ORG: infrawatch
+        OPERATOR_SDK: "{{ playbook_dir }}/working/operator-sdk-v1.5.0"
```

Comment:
If you set the version as a fact in the defaults you could use it here and in the other plays to sync the version across. I'm not sure I like seeing the version hard coded across so many locations.